datadog_checks_dev/datadog_checks/dev/tooling/e2e/run.py

```diff
@@ -30,9 +32,9 @@ def _execute(check, command, env_vars):
 def start_environment(check, env):
     env_vars = {E2E_TEAR_DOWN: 'false', 'PYTEST_ADDOPTS': '--exitfirst'}
     if file_exists(get_hatch_file(check)):
-        command = f'hatch env run --env {env} test'
+        command = [sys.executable, "-m", 'hatch', 'env', 'run', '--env', env, 'test']
     else:
-        command = f'tox --develop -e {env}'
+        command = [sys.executable, "-m", "tox", '--develop', '-e', env]
```

Comment:
Could you also update https://github.com/DataDog/integrations-core/blob/master/datadog_checks_dev/datadog_checks/dev/tooling/e2e/run.py#L48-L52 ?

Reply:
woah, how did I not see that with grep?

Reply:
Seems like `eyes` > `grep` then 😁